### DISCUSSION TITLE
Welcome to instacalc Discussions!

### DISCUSSION THREAD

**Discussion by kazad** [kazad/instacalc#1] [Announcements]
## 👋 Hey!

I'm using github discussions to host the discussion forum. Feel free to ask questions, share ideas, or chat with others. Enjoy your calc'in!

**Comment by jimmyrrr** [comment]
Can you make the expression edit field a little longer? Or can be optional selectable length?

**Reply by kazad** [reply]
The text edit field now has an icon to expand as it gets full (see https://github.com/kazad/instacalc/discussions/72)

**Comment by kazad** [comment]
Thanks for the suggestion -- I'd like to to have the text edit field auto-expand as you type. Right now it's hard to write longer expressions. Appreciate the feedback!

**Comment by lsholguin** [comment]
Wow!  Thank you for the update!

Kind regards.-

**Comment by jimmyrrr** [comment]
Thank you for the update.
This will do
[image: image.png]
I also prefer a wider field as my monitor is wide
[image: image.png]
Best regards,
Jeremy

Kalid Azad ***@***.***> 于2024年3月18日周一 15:42写道：

**Reply by kazad** [reply]
Thanks! Could you upload the images again, I think github may not have received them. Appreciate it!

**Comment by jimmyrrr** [comment]
download this
https://tinyjpg.com/backend/opt/output/v1xc9v1s0xrkjcj0g71sz34740dpecp0/wideneditfield.png

Kalid Azad ***@***.***> 于2024年3月20日周三 14:10写道：

**Reply by kazad** [reply]
Thanks, I see what you mean. Currently, the input column/output column are about 50%, but perhaps a draggable divider (or a "flex" size that grows to the contents) may help set a good size. Appreciate the suggestion!

**Comment by jimmyrrr** [comment]
Just found there is unfriendly delay in edit field , you cannot see what
you entered immediately, about 1 second delay at least, it does not happen
in my other programs including  any google page edit field .

Kalid Azad ***@***.***> 于2024年3月23日周六 12:35写道：

**Reply by kazad** [reply]
Thanks for the report. I can't seem to reproduce it, but I'll keep an eye on logs to see if there is a slowdown. There haven't been any recent changes that would impact perf (to my knowledge). Let me know if it's still happening in the next few days, after a browser reload. Thanks!

**Comment by jimmyrrr** [comment]
my issue then, never mind, you are the best already!

Kalid Azad ***@***.***> 于2024年9月21日周六 02:43写道：

**Comment by kazad** [comment]
No worries, I'll keep an eye out for any reports!
